src/main/java/uk/gov/hmcts/darts/common/entity/Courthouse.java

```diff
@@ -24,31 +22,17 @@ public class Courthouse {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Integer id;
 
-    @Column(name = "r_courthouse_object_id", length = 16)
-    private String legacyObjectId;
+    @Column(name = "code")
+    private short code;
 
-    @Column(name = "c_code", length = 32)
-    private String code;
+    @Column(name = "courthouse_name")
+    private String courthouseName;
 
-    @Column(name = "c_id", length = 32)
-    private String idString;
+    @CreationTimestamp
+    @Column(name = "created_date_time")
+    private Timestamp createdDateTime;
 
-    @Column(name = "c_alias_set_id", length = 16)
-    private String aliasSetIdString;
-
-    @Column(name = "r_version_label", length = 32)
-    private String legacyVersionLabel;
-
-    @Column(name = "i_superseded")
-    private Boolean superseded;
-
-    @Version
-    @Column(name = "i_version_label")
```

Comment:
@cmagowan-hmcts I think we will need the Optimistic Locking version column adding to our entity, not sure why this is missing now?

`i_version                SMALLINT`

Reply:
I found a message from a chat I had with Barry on Teams 25/05
"The only tables i've removed superceded and version labels from , is where we have no evidence of updates arising over lifetime of the data, such as with courthouse table."

So if we are proposing to manage the Courthouse data in Modernised DARTS then sure we need to add the version column back